src/caret_analyze/plot/timeseries/frequency_timeseries.py

```diff
@@ -139,19 +139,14 @@ def row_filter_communication(record) -> bool:
     def _get_timestamp_range(
         timeseries_records_list: list[RecordsInterface]
     ) -> tuple[int, int]:
-        first_timestamps = []
-        last_timestamps = []
+        all_timestamps = []
         for records in timeseries_records_list:
             if len(records) == 0:
                 continue
-            first_timestamp = records.get_column_series(records.columns[0])[0]
-            if isinstance(first_timestamp, int):
-                first_timestamps.append(first_timestamp)
-            last_timestamp = records.get_column_series(records.columns[0])[-1]
-            if isinstance(last_timestamp, int):
-                last_timestamps.append(last_timestamp)
-
-        if len(first_timestamps) == 0 or len(last_timestamps) == 0:
+            timestamps = records.get_column_series(records.columns[0])
+            all_timestamps.extend(Util.filter_items(lambda x, : isinstance(x, int), timestamps))
+
+        if len(all_timestamps) == 0 or len(all_timestamps) == 0:
```

Comment:
Minor fix of duplicated code.

```suggestion
        if len(all_timestamps) == 0:
```

Reply:
Fixed in [b78e69e](https://github.com/tier4/caret_analyze/pull/455/commits/b78e69ec501fdf07ea57a217b4b8931a4b23088f).